go.mod

```diff
@@ -43,6 +43,8 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+replace vbom.ml/util => github.com/fvbommel/util v0.0.2
```

Comment:
I think we may need to reference the same commit that we referenced before instead of v0.0.2, since this appears to change some dependencies.

Can you try referencing v0.0.0-20180919145318-efcd4e0f9787 instead of v0.0.2? Or the other one from go.sum. I don't know which one is correct.

Reply:
Done! :)